terasoluna-gfw-functionaltest-web/src/main/resources/META-INF/spring/spring-mvc-common.xml

```diff
@@ -79,31 +79,19 @@
       <mvc:mapping path="/**" />
       <mvc:exclude-mapping path="/resources/**" />
       <mvc:exclude-mapping path="/**/*.html" />
-      <mvc:exclude-mapping path="/codelist/fallbackToEn/**" />
       <mvc:exclude-mapping path="/codelist/fallbackToFr/**" />
       <mvc:exclude-mapping path="/codelist/noPattern/**" />
       <mvc:exclude-mapping path="/codelist/jdbcCodeListTestDBError/**" />
       <bean class="org.terasoluna.gfw.web.codelist.CodeListInterceptor">
         <property name="codeListIdPattern" value="CL_.+" />
       </bean>
     </mvc:interceptor>
-    <mvc:interceptor>
-      <mvc:mapping path="/codelist/fallbackToEn/**" />
-      <mvc:exclude-mapping path="/resources/**" />
-      <mvc:exclude-mapping path="/**/*.html" />
-
-      <bean class="org.terasoluna.gfw.web.codelist.CodeListInterceptor">
-        <property name="codeListIdPattern" value="CL_.+" />
-        <property name="fallbackTo" value="en" />
-      </bean>
-    </mvc:interceptor>
     <mvc:interceptor>
       <mvc:mapping path="/codelist/fallbackToFr/**" />
       <mvc:exclude-mapping path="/resources/**" />
       <mvc:exclude-mapping path="/**/*.html" />
       <bean class="org.terasoluna.gfw.web.codelist.CodeListInterceptor">
-        <property name="codeListIdPattern" value="CL_.+" />
-        <property name="fallbackTo" value="fr" />
+        <property name="codeListIdPattern" value="CLFR_.+" />
       </bean>
     </mvc:interceptor>
```

Comment:
- [ ] You no longer need to use `CodeListInterceptor` differently to change the fallback locale. Do you need to define multiple fallback `CodeListInterceptors`?

Reply:
Modified by ec1c229ced6a7858f02e35d02e352e85eac81f59
I did not need it, so I erased it.